### DISCUSSION TITLE
SSH?

### DISCUSSION THREAD

**Discussion by Oxize** [vercel/hyper#7428] [General]
Maybe noob question. But i though this Hyper was an SSH Client. I downloaded it for windows, and just seeing a dos box comming up. Isnt it an SSH client for windows?

**Comment by lnfel** [comment]
As the description of the repo Hyper is, "**A terminal built on web technologies**". Not an ssh client, it more similar to windows cmd and macos terminal and you can run `ssh` on a cmd/terminal provided that your environment is configured to do so.

In windows it would be something like what microsoft is showing in their docs:
https://learn.microsoft.com/en-us/windows/terminal/tutorials/ssh

Try running `ssh` on cmd or hyper to see if your ssh is available, you'll see more info by reading the documentation referenced on the link above.
